packages/dd-trace/src/priority_sampler.js

```diff
@@ -206,4 +208,4 @@ function hasOwn (object, prop) {
   return Object.prototype.hasOwnProperty.call(object, prop)
 }
 
-module.exports = PrioritySampler
+module.exports = { PrioritySampler, hasOwn }
```

Comment:
I think that `hasOwn` is not related with `priority_sampler.js` file, it is just a utils method to call to `hasOwnProperty`, you could use `hasOwnProperty` where you need it, or extract `hasOwn` function to utils file, but I think that it is not enough related with this file to export it here.

Reply:
I had doubts about it too, was tempted to implement `_hasOwn` in `PrioritySampler` to avoid the export but I could move it to `utils.js` file...

Reply:
it might makes sense to have it in `packages/dd-trace/src/util.js`